### PR TITLE
updateAnswer mutation can use api accesstoken

### PR DESCRIPTION
**Pull request by kycarr**

### CODE REVIEW

File: node/src/gql/mutation/me/update-answer.ts [hunk 1]
```diff
@@ -20,17 +20,20 @@ import {
...
 }
 
-export const AnswerInputType = new GraphQLInputObjectType({
-  name: 'AnswerInputType',
+export const UpdateAnswerInputType = new GraphQLInputObjectType({
+  name: 'UpdateAnswerInputType',
   fields: () => ({
     transcript: { type: GraphQLString },
     status: { type: GraphQLString },
+    recordedAt: { type: GraphQLString },
```
Comment (beatthat) [inline node/src/gql/mutation/me/update-answer.ts:36]: usually I would say client is not allowed to set timestamps, and the server should do that instead. Any reason we are enabling it here?

File: node/src/gql/mutation/me/update-answer.ts [hunk 2]
```diff
@@ -53,7 +56,10 @@ export const updateAnswer = {
     if (!mentor) {
       throw new Error(`no mentor found for id '${args.mentorId}'`);
     }
-    if (`${context.user._id}` !== `${mentor.user}`) {
+    if (
+      `${context.user._id}` !== `${mentor.user}` &&
+      context.user._id !== requireEnv('API_SECRET')
```
Comment (beatthat) [inline node/src/gql/mutation/me/update-answer.ts:61]: can we get this to work without setting the API_SECRET as `context.user._id`? That seems like it's asking for trouble

**Review by beatthat — CHANGES_REQUESTED** [review]
looks good. a few questions/comments inline

File: node/src/gql/mutation/me/update-answer.ts (hunk 1 above)
Comment (kycarr) [reply node/src/gql/mutation/me/update-answer.ts:36]: mentor-upload is calling this too right? wouldn't we want mentor-upload to set recordedAt? or should there be two separate update answers.

Comment (beatthat) [reply node/src/gql/mutation/me/update-answer.ts:36]: ok, so idea is to have distinct db fields for `recordedAt` vs `updatedAt` which might just be the transcription? Who is using this data?

Comment (kycarr) [reply node/src/gql/mutation/me/update-answer.ts:36]: The client is using it to check whether or not they should fetch the recorded video for a mentor (using the mentor id and question id)

Comment (beatthat) [reply node/src/gql/mutation/me/update-answer.ts:36]: We can leave for now if it's it makes the demo work, but when there is really video, the client should be basing that logic on the presence of actual media urls, probably something like

```json
{
  "transcript": "a b c",
  "media" [
    {
       "type": "video-mobile",
       "url": "..."
    },
    {
       "type": "video-web",
       "url": "..."
    }
  ]
}

**Review by beatthat — APPROVED** [review]

**Pull request merged**